README.md

```diff
@@ -29,10 +29,10 @@ With Gradle from repo.spring.io or Maven Central repositories (stable releases o
     }
 
     dependencies {
-      //compile "io.projectreactor:reactor-core:3.3.6.BUILD-SNAPSHOT"
-      //testCompile("io.projectreactor:reactor-test:3.3.6.BUILD-SNAPSHOT")
-      compile "io.projectreactor:reactor-core:3.3.5.RELEASE"
-      testCompile("io.projectreactor:reactor-test:3.3.5.RELEASE")
+      //compile "io.projectreactor:reactor-core:3.4.1-SNAPSHOT"
```

Comment:
shouldn't we use `3.4.0-SNAPSHOT` here? 3.4.0 is not released yet, so it can be a bit misleading

Reply:
yeah we can. first release won't be able to use the bumpVersionInReadme task, so I had the change upfront, but we'll need to remember to do it manually instead.